content/posts/2022-features.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "2022 for Packit"
+date: 2023-01-28T11:58:50+01:00
+---
+
+## Packit project in 2022
+
+As you will see in the following paragraphs, the year 2022 was really fruitful for the Packit project. Without further ado, let’s take a look at what the Packit team accomplished last year:
+
+## Fedora automation
+
+Packit team has made a huge improvement in downstream automation. At the beginning of the year, we [finished the workflow](https://packit.dev/posts/downstream-automation/) and you are now able to use Packit to get your release from upstream via dist-git and Koji to Bodhi. As usual, you can pick just what you need. This workflow consists of three jobs:
+
+- propose-downstream: as a reaction to an upstream release, the source archive is saved to a lookaside cache, spec-file is updated and sent as a pull request to Fedora dist-git.
+- koji-build: as a reaction to a new dist-git commit, a new Koji build is triggered (you can specify allowed authors of a commit or merged pull request).
+- bodhi-update: as a reaction to a successfully finished Koji build, a new Bodhi update is created
+
+TODO: imag (bodhi-update?)
+
+But that wasn’t all for this year. At the very end of the year, the Packit team implemented an alternative to the `propose-downstream` job that we call `pull-from-upstream`. The logic of the job is the very same: the source archive is saved to a lookaside cache, spec-file is updated and sent as a pull request to Fedora dist-git. The only – and main – difference is that the job is defined downstream (in the main dist-git branch) so you don’t need to install Packit in the upstream repository. The information about a new release is received from the [Upstream Release Monitoring](). The `pull-from-upstream` job is mainly targeted to the Fedora maintainers without upstream access or with upstream not being supported by Packit. (This job works with any upstream using GIT.) The setup is nicely described in [this blog post](). And if you want a dedicated documentation page for the Fedora downstream automation, look at https://packit.dev/docs/fedora-releases-guide/.
+
+TODO: image (pull-from-upstream)
+
+And that’s still not all, we’ve also added `propose-downstream` to our dashboard. You can now the jobs in the Pipelines view. Also, logs can be checked on a detail page (that can be accessed from a status of a release commit).
+
+TODO: image (pipelines with propose-downstream>)
+
+Still not convinced we’ve done a lot? When using various downstream jobs, you can still hit an issue now and then. (messages from Fedora infrastructure can get lost or some flake error can occur.) Packit tries to retry automatically to overcome temporary issues, but sometimes it’s not enough or there is a real problem that needs to be fixed elsewhere. We can’t resolve the real problems for you, but we can help let you retry the job when needed. And you have two places where to do that.
```

Comment:
```suggestion
Still not convinced we’ve done a lot? When using various downstream jobs, you can still hit an issue now and then. (messages from Fedora infrastructure can get lost or a network flake can occur.) Packit automatically retries tasks to overcome temporary issues, but sometimes it’s not enough or there is a real problem that needs to be fixed elsewhere. We can’t resolve the real problems for you, but we can help let you retry the job when needed. And you have two places where to do that.
```

Reply:
I am not sure if the issues are just with the network..;)

Reply:
yes, that's true, in that case, I'd use "flake" or "intermittent error"; "flake error" term is not being used commonly